src/screens/send/components/add-address/add-address.tsx

```diff
@@ -0,0 +1,347 @@
+import React from 'react';
+import { View, TextInput, TouchableOpacity, Platform } from 'react-native';
+import stylesProvider from './styles';
+import { withTheme, IThemeProps } from '../../../../core/theme/with-theme';
+import { Text } from '../../../../library';
+import { smartConnect } from '../../../../core/utils/smart-connect';
+import { translate } from '../../../../core/i18n';
+import { AccountList } from '../account-list/account-list';
+import { IAccountState } from '../../../../redux/wallets/state';
+import { AddressBook } from '../address-book/address-book';
+import { IContactState, IContactsState } from '../../../../redux/contacts/state';
+import { formatAddress } from '../../../../core/utils/format-address';
+import { Icon } from '../../../../components/icon';
+import { ICON_SIZE } from '../../../../styles/dimensions';
+import { Dialog } from '../../../../components/dialog/dialog';
+import { getBlockchain } from '../../../../core/blockchain/blockchain-factory';
+import {
+    ResolveTextCode,
+    ResolveTextType,
+    ResolveTextError,
+    ChainIdType,
+    Blockchain
+} from '../../../../core/blockchain/types';
+import { addContact } from '../../../../redux/contacts/actions';
+import { connect } from 'react-redux';
+import { QrModalReader } from '../../../../components/qr-modal/qr-modal';
+
+export interface IExternalProps {
+    account: IAccountState;
+    accounts: IAccountState[];
```

Comment:
accounts can be taken from redux directly, and remove from send screen.
If if other redux props can be move here only